### ISSUE TITLE
HELP Select by cell and not row

### ISSUE THREAD

**Issue by Tiliaa** [tabalinas/jsgrid#976]
Hello Tabalinas !
First of all, thanks a lot for the JsGrid tool, really simple to use and powerful ! I'm not sure if what I need to do is actuallys upported by jsGrid, that's why I create the subject.
What I want to do is the same as this subject https://github.com/tabalinas/jsgrid/issues/458 except i need to make each field editable or not depending on separate values.
Example : 
         | value | percentage | value_editable | percentage editable |
line 1 |   2     |  5               |        true         | false                       |
line 3 |   8     |  8               |       false         | true                        |

Here, I need to make value editable and percentage read only for line 1, and value read only and percentage editable for line 2. Would that be possible ? And how ?

Thanks a lot,
Tilia

**Comment by tabalinas** [comment]
Well, you need a condition which is going to define non-editable field. The principle is still the same as here https://github.com/tabalinas/jsgrid/issues/458#issuecomment-251808862.
Please, clarify why this doesn't work for your case.

**Comment by Tiliaa** [comment]
Hello,
Thank you for your answer. This doesn't work because I have a contradiction. In my exemple, for line 1, i have value_editable = true that make the field "value" editable and percentage_editable = false that make the field "percentage" read only. 
The problem is that "value" and "percentage" are selected by the same object "item", in the solution in #458 you only make the whole row editable or not (and I would need to make only one cell editable on this line).
I hope it's more clear now.

**Comment by tabalinas** [comment]
right, jsgrid supports editing for entire row. The possible workaround is changing the item on row click, so that you could show editing control only for the clicked.
